Gems/WhiteBox/Code/Tests/WhiteBoxPhysicsTest.cpp

```diff
@@ -47,7 +47,7 @@ namespace UnitTest
 
     void EditorWhiteBoxPhysicsTestEnvironment::AddGemsAndComponents()
     {
-        AddDynamicModulePaths({"PhysX.Editor.Gem"});
+        AddDynamicModulePaths({"PhysX.Editor"});
```

Comment:
The [output name](https://github.com/o3de/o3de/blob/7b659cd0fbf274254ab19d658840b98ae17b6228/Gems/PhysX/Code/CMakeLists.txt#L126) for this gem is `PhysX.Editor.Gem`. This was needed for the client gem module because on some platforms, the name of the 3rdParty lib and the gem lib ended up being the same. It's not needed for tools but it looks like it was updated anyway.

Reply:
I'll change it back.
It actually is trying to load the PhysX.Editor.Gem.dll file during the test.